mode/smalltalk/smalltalk.js

```diff
@@ -120,6 +122,17 @@ CodeMirror.defineMode('smalltalk', function(config) {
 
     if (aChar === '|') {
       token.context = context.parent;
+      var str = stream.string;
```

Comment:
I'm not a JS developer by trade, but I am learning. I'll give the RegEx a shot. Basically I'm trying to find whether or not there are additional variables in a block, so [:var1 | |var2 var3| ...]